packages/salesforcedx-vscode-soql/test/jest/editor/soqlEditorProvider.test.ts

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import {
+  BUILDER_VIEW_TYPE,
+  SOQL_BUILDER_WEB_ASSETS_PATH
+} from '../../../src/constants';
+import { HtmlUtils } from '../../../src/editor/htmlUtils';
+import { SOQLEditorInstance } from '../../../src/editor/soqlEditorInstance';
+import { SOQLEditorProvider } from '../../../src/editor/soqlEditorProvider';
+import { nls } from '../../../src/messages';
+import { channelService, isDefaultOrgSet } from '../../../src/sf';
+
+jest.mock('../../../src/sf', () => ({
+  ...jest.requireActual('../../../src/sf'),
+  isDefaultOrgSet: jest.fn()
+}));
```

Comment:
interesting...is this to get around the fact that modules is a big bundle of random exports? 

It's not necessarily better, but is this the same as 
```
import * as sf from '../.../../src/sf';

beforeEach(() => {
  isDefaultOrgSetSpy = jest.spyOn(sf, 'isDefaultOrgSet');
});
```

totally fine to 🤷

Reply:
I like this better, I had forgotten this could be done though 😆 thanks